### PR TITLE
nano: add new command for backup

### PR DESCRIPTION
**Pull request by Ray6464**
- [ ] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/common/nano.md [hunk 1]
```diff
@@ -18,3 +18,7 @@
 - Indent new lines to the previous lines' indentation:
 
 `nano -i {{filename}}`
+
+- Before saving, back up the previous version of the file with name = "currentName+tilde" (useful for quick edits)
```
Comment (navarroaxel) [inline pages/common/nano.md:22]: Maybe this is better:
```suggestion
- Back up the previous version as `{{filename}}~` when save:
```
or
```suggestion
- Before saving, back up the previous version as `{{filename}}~`:
```

Comment (sbrl) [reply pages/common/nano.md:22]: Maybe `on save` instead of `when save` there @navarroaxel? Other than that, either is fine :+1:

Comment (Ray6464) [reply pages/common/nano.md:22]: If we're going that way, how about this?:
```suggestion
- Before saving, back up the previous version of the file with name = "currentName+tilde" (useful for quick edits)
+ Before modification, backup separately as `{{currentFileName}}~`
```

Comment (navarroaxel) [reply pages/common/nano.md:22]: @sbrl ?

Comment (sbrl) [reply pages/common/nano.md:22]: That would work too! Let's do that. Although, we prefer `snake_case` to `pascalCase`:
```suggestion
- Before modification, backup separately as `{{current_file_name}}~`:
```

**Review by navarroaxel — APPROVED** [review]

**Review by sbrl — APPROVED** [review]
Thanks, @Ray6464!

**Pull request merged**